src/Build/Logging/BinaryLogger/Postprocessing/IBuildFileReader.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Build.Logging;
+
+public interface IBuildFileReader
```

Comment:
What is the advantage of putting this in its own interface?

Reply:
I'm not strongly opinionated - so happy to merge those if there is anyone with strong opinion :-), but I lean towards splitting separate concerns to separate interfaces.

The current definition:

```csharp
    public interface IBuildEventArgsReaderNotifications :
        IBuildEventStringsReader,
        IBuildFileReader,
        IBinaryLogReaderErrors
    { }
```

Only `IBuildEventArgsReaderNotifications` can be kept and hold the events of all 3.
On the other hand each of those 3 have different concern.


What's your thoughts on this?

Reply:
I don't have super strong feelings, but was somewhat surprised to see the several new ones.

I guess the question I'd like to consider is "will users care about the distinction?"

Reply:
Right - probably superfluous. I'm merging those 3 into the parent one.